com.tibco.utils.wrapper.jaxb.TestJaxbHelper Unit test failing

[INFO] Running com.tibco.utils.wrapper.jaxb.TestJaxbHelper
[ERROR] Tests run: 5, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 0.903 s <<< FAILURE! -- in com.tibco.utils.wrapper.jaxb.TestJaxbHelper
[ERROR] com.tibco.utils.wrapper.jaxb.TestJaxbHelper.testSimpleLoad -- Time elapsed: 0.649 s <<< FAILURE!
java.lang.AssertionError
	at org.junit.Assert.fail(Assert.java:87)
	at org.junit.Assert.assertTrue(Assert.java:42)
	at org.junit.Assert.assertNotNull(Assert.java:713)
	at org.junit.Assert.assertNotNull(Assert.java:723)
	at com.tibco.utils.wrapper.jaxb.TestJaxbHelper.testSimpleLoad(TestJaxbHelper.java:65)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:316)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeWithRerun(JUnit4Provider.java:240)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:214)
	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:155)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
	at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)

[INFO] Running com.tibco.utils.Bw5_utils.AppTest
[INFO] Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.001 s -- in com.tibco.utils.Bw5_utils.AppTest
[INFO] Running com.tibco.sonar.plugins.bw6.check.process.ForEachGroupCheckTest
parseProcessName - START
Nodeset size founded: 1

--- a/src/test/resources/xml/ext-archetype-metadata.sample0.xml
+++ b/src/test/resources/xml/ext-archetype-metadata.sample0.xml
@@ -1,19 +1,19 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
 <ns3:extarchetype-descriptor
-	xmlns:ns2="http://www.tibco.com/xmlns/extarchetype/validation"
-	xmlns:ns3="http://www.tibco.com/xmlns/extarchetype/metadata">
-	<requiredProperties>
-		<requiredProperty key="param1">
+		xmlns:ns2="http://www.tibco.com/xmlns/extarchetype/validation"
+		xmlns:ns3="http://www.tibco.com/xmlns/extarchetype/metadata">
+	<ns3:requiredProperties>
+		<ns3:requiredProperty key="param1">
 			<ns2:validationRule>validationRule1</ns2:validationRule>
-			<source>deduction</source>
-		</requiredProperty>
-		<requiredProperty key="param2">
+			<ns3:source>deduction</ns3:source>
+		</ns3:requiredProperty>
+		<ns3:requiredProperty key="param2">
 			<ns2:validationRule>validationRule2</ns2:validationRule>
-			<source>global</source>
-		</requiredProperty>
-		<requiredProperty key="param3">
+			<ns3:source>global</ns3:source>
+		</ns3:requiredProperty>
+		<ns3:requiredProperty key="param3">
 			<ns2:validationRule>validationRule3</ns2:validationRule>
-			<source>input</source>
-		</requiredProperty>
-	</requiredProperties>
+			<ns3:source>input</ns3:source>
+		</ns3:requiredProperty>
+	</ns3:requiredProperties>
 </ns3:extarchetype-descriptor>

